qiskit/primitives/base/base_estimator.py

```diff
@@ -83,69 +80,6 @@
     job_result = job2.result()
     print(f"The primitive-job finished with result {job_result}")
 
-
-========================
-Overview of EstimatorV1
-========================
-
-Estimator class estimates expectation values of quantum circuits and observables.
-
-An estimator is initialized with an empty parameter set. The estimator is used to
-create a :class:`~qiskit.providers.JobV1`, via the
-:meth:`qiskit.primitives.Estimator.run()` method. This method is called
-with the following parameters
-
-* quantum circuits (:math:`\psi_i(\theta)`): list of (parameterized) quantum circuits
-  (a list of :class:`~qiskit.circuit.QuantumCircuit` objects).
-
-* observables (:math:`H_j`): a list of :class:`~qiskit.quantum_info.SparsePauliOp`
-  objects.
-
-* parameter values (:math:`\theta_k`): list of sets of values
-  to be bound to the parameters of the quantum circuits
-  (list of list of float).
-
-The method returns a :class:`~qiskit.providers.JobV1` object, calling
-:meth:`qiskit.providers.JobV1.result()` yields the
-a list of expectation values plus optional metadata like confidence intervals for
-the estimation.
-
-.. math::
-
-    \langle\psi_i(\theta_k)|H_j|\psi_i(\theta_k)\rangle
-
-Here is an example of how the estimator is used.
-
-.. code-block:: python
-
-    from qiskit.primitives import Estimator
-    from qiskit.circuit.library import RealAmplitudes
-    from qiskit.quantum_info import SparsePauliOp
-
-    psi1 = RealAmplitudes(num_qubits=2, reps=2)
-    psi2 = RealAmplitudes(num_qubits=2, reps=3)
-
-    H1 = SparsePauliOp.from_list([("II", 1), ("IZ", 2), ("XI", 3)])
-    H2 = SparsePauliOp.from_list([("IZ", 1)])
-    H3 = SparsePauliOp.from_list([("ZI", 1), ("ZZ", 1)])
-
-    theta1 = [0, 1, 1, 2, 3, 5]
-    theta2 = [0, 1, 1, 2, 3, 5, 8, 13]
-    theta3 = [1, 2, 3, 4, 5, 6]
-
-    estimator = Estimator()
-
-    # calculate [ <psi1(theta1)|H1|psi1(theta1)> ]
-    job = estimator.run([psi1], [H1], [theta1])
-    job_result = job.result() # It will block until the job finishes.
-    print(f"The primitive-job finished with result {job_result}"))
-
-    # calculate [ <psi1(theta1)|H1|psi1(theta1)>,
-    #             <psi2(theta2)|H2|psi2(theta2)>,
-    #             <psi1(theta3)|H3|psi1(theta3)> ]
-    job2 = estimator.run([psi1, psi2, psi1], [H1, H2, H3], [theta1, theta2, theta3])
-    job_result = job2.result()
-    print(f"The primitive-job finished with result {job_result}")
```

Comment:
Why are we removing the V1 api docs? This documentation is valuable until the official removal of the class (we keep the API docs during the deprecation period too, and this is even pre-deprecation period).

Reply:
This was removed as part of a process to reshuffle the order of the page, but this was happening on the 1st when there was a ban on unnecessary commits to the repo. I'll commit sometime soon, sorry for the confusion.